### PR TITLE
Dockerfile.nv: two apt-get update was missing

### PR DESCRIPTION
**Pull request by fcqpl**
Error when tried to update OME:
```
Get:133 http://archive.ubuntu.com/ubuntu bionic/universe amd64 tcl amd64 8.6.0+9 [5146 B]
Get:134 http://archive.ubuntu.com/ubuntu bionic/main amd64 zlib1g-dev amd64 1:1.2.11.dfsg-0ubuntu2 [176 kB]
Get:135 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 uuid-dev amd64 2.31.1-0.4ubuntu3.7 [33.2 kB]
Fetched 71.7 MB in 6s (12.9 MB/s)
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1-1ubuntu2.1~18.04.9_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/o/openssl/openssl_1.1.1-1ubuntu2.1~18.04.9_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/l/linux/linux-libc-dev_4.15.0-147.151_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/c/cmake/cmake-data_3.10.2-1ubuntu2.18.04.1_all.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/c/curl/libcurl4_7.58.0-2ubuntu3.13_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/c/cmake/cmake_3.10.2-1ubuntu2.18.04.1_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/c/curl/curl_7.58.0-2ubuntu3.13_amd64.deb  404  Not Found [IP: 91.189.88.142 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
The command '/bin/sh -c apt-get -y install build-essential nasm autoconf libtool zlib1g-dev tclsh cmake curl pkg-config bc uuid-dev apt-utils' returned a non-zero code: 100
ERROR: Service 'ovenmediaengine' failed to build
```

### CODE REVIEW

**Comment by fcqpl** [comment]
Och, it's look like similar pull request is exist: https://github.com/AirenSoft/OvenMediaEngine/pull/307#issue-595576852

**Comment by basisbit** [comment]
`apt update` is only needed once, not for every step of the container build?

**Comment by basisbit** [comment]
So, this problem happens when someone has an old docker container build, changes something and then docker only does a partial container rebuild. Then the apt package info cache can be old, and this issue occurs. This pull request will solve the symptoms for most cases, but doesn't fix the problem that the user has a container build layer with partially old (and potentially very old) software, which might be missing critical security updates. This is a common problem with Docker when people don't regularly do full rebuilds of their docker containers. They then run old and insecure software with known problems, but they themself don't even realize it, because for them it is just "a container".

TLDR: when using Docker, the user still has to make sure to run current software, just like on any linux server.

**Comment by basisbit** [comment]
It might be a better idea to not merge this PR, so the user actually notices that they are running old suff, and then maybe improve documentation so that the user knows how to rebuild the whole container (`docker-compose build --no-cache`)
